tools/apidiff/Makefile

```diff
@@ -245,6 +264,18 @@ endif
 	@echo "@MonkeyWrench: AddFile: $(CURDIR)/api-diff.html"
 	@# remove empty files so they're not uploaded
 	@-find $(CURDIR)/diff -size 0 | xargs rm
+ifdef INCLUDE_IOS
```

Comment:
I think this can be improved to make it more parallelizable, by declaring the *-markdown targets like this:

```makefile
ifdef INCLUDE_IOS
ios-markdown: merger.exe $(foreach file,$(IOS_ASSEMBLIES),$(APIDIFF_DIR)/diff/xi/$(file).md)
    $(Q) $(SYSTEM_MONO) --debug merger.exe Xamarin.iOS $(CURDIR)/diff/xi/Xamarin.iOS/ ios
else
ios-markdown: ; @true
endif

ifdef INCLUDE_TVOS
    ...
endif

...
```

and then here do:

```makefile
    $(MAKE) -j8 ios-markdown tvos-markdown watchos-markdown macos-markdown
```

I'm not sure how much of a difference it will make though.